solidity/.solhint.json

```diff
@@ -6,6 +6,7 @@
         "no-global-import": "off",
         "no-inline-assembly": "off",
         "one-contract-per-file": "off",
-        "ordering": "off"
+        "ordering": "off",
+        "function-max-lines": "off"
```

Comment:
this could be useful to reduce the stake so we avoid --via-ir

Reply:
You mean the stack?
Yes and no. The reason I turned it off is that there will be a decent number of methods that end up having more than 50% of the assembly block simply be import boilerplate.
For example, https://github.com/spaceandtimelabs/sxt-proof-of-sql/pull/546/files#diff-aa3fe37b7bb1dfcabed7a7cb222af92433c92df7a65ca4a690cf0fce32dc8c0eR285-R361 is huge.
It also happens to be nearing the stack limit because there are a ton of parameters, but the actual code is only 16 lines long.